plenum/test/test_bootstrapping.py

```diff
@@ -49,6 +47,5 @@ def testConnectWithoutKeySharingFails(tdir_for_func):
         with TestNodeSet(names=nodeNames, tmpdir=tdir_for_func,
                          keyshare=False) as nodes:
             with Looper(nodes) as looper:
-                    looper.run(
-                            checkNodesConnected(nodes, NOT_CONNECTED))
+                looper.runFor(2)
```

Comment:
Why did we remove check for connection and use hard-coded timeout (BTW all timeouts needs to be set in waits.py)?